src/applications/disability-benefits/all-claims/pages/medicalCareHistory.js

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+
+const medicalCareDescription = (
+  <div>
+    <h5>Medical care</h5>
+    <p>
+      During the last 12 months, have you been under a doctor's care or
+      hospitalized for these disabilities?
+    </p>
+  </div>
+);
+
+export const uiSchema = {
+  'ui:title': ' ',
+  'ui:description': medicalCareDescription,
+  careQuestion: {
+    'ui:title': ' ',
+    'ui:widget': 'radio',
```

Comment:
Just FYI, there is a `yesNo` widget for this which will give us a boolean back instead of checking against a yes or no string.